velox/docs/functions/spark/string.rst

```diff
@@ -241,15 +241,18 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
     Returns a map by splitting ``string`` into entries with ``entryDelimiter`` and splitting
     each entry into key/value with ``keyValueDelimiter``.
-    ``entryDelimiter`` and ``keyValueDelimiter`` must be constant strings with single ascii
-    character. Allows ``keyValueDelimiter`` not found when splitting an entry. Throws exception
-    when duplicate map keys are found for single row's result, consistent with Spark's default
-    behavior. ::
+    The entryDelimiter and keyValueDelimiter can be multiple characters, 
+    can also be regular expressions, and support Unicode encoded strings.
```

Comment:
> The entryDelimiter and keyValueDelimiter can be multiple characters, 
    can also be regular expressions, and support Unicode encoded strings.

Document suggestion:

> \``entryDelimiter\`` and \``keyValueDelimiter\`` are UTF-8 characters and treated as regular expressions. 

Could you check the behavior of invalid UTF-8 characters?